LCD_digits/test/display.js

```diff
@@ -0,0 +1,41 @@
+let chai = require('chai');
+let should = chai.should();
+let displayer = require('../displayer');
```

Comment:
there's no module called "displayer" located in the above directory. Same for digits.